package.json

```diff
@@ -104,7 +104,7 @@
     "check:eslint": "eslint -v && eslint --max-warnings=0 --ext '.js,.ts' src test",
     "check:tsd": "tsd --version && tsd",
     "check:dts": "./node_modules/typescript/bin/tsc --noEmit mongodb.d.ts && tsd",
-    "check:test": "mocha --file test/tools/runner --recursive test/functional test/integration",
+    "check:test": "mocha --file test/tools/runner --recursive test/integration",
```

Comment:
We can drop the recursive flag from here, I think at some point it was needed for some runs and not others but its enabled in our mocharc